components/uniset/src/uniset_const.rs

```diff
@@ -0,0 +1,1928 @@
+#![allow(clippy::unreadable_literal, dead_code)]
+
+use crate::uniset::UnicodeSet;
+
+/// Upper
+pub fn upper() -> UnicodeSet {
```

Comment:
I don't want to check in this file in its current state.  Please do one of the following:

1. Add a dummy DataProvider argument to all of the functions.
2. Option 1, plus implement DataProvider for ppucd and use it.  (my desired long-term solution)
3. Rename this file in a way to emphasize that these sets are purely for testing purposes and will be removed in a future release, definitely before v1.

Reply:
Done.